public/style.css

```diff
@@ -155,7 +155,7 @@ body {
   border-radius: 50%;
 }
 .container .left .people {
-  margin-left: -1px;
+  margin-left: 25px;
```

Comment:
This doesn't look correct. Doing this moves the entire div the right which can be seen by the right line (border) that comes in the message area. Instead doing `padding-left: 20%` seems proper as the div then remains in the correct place and the room list gets smaller and moves to the right.
Also prefer % over px as it makes the design adapt different screen sizes.